src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -222,20 +223,21 @@ protected static LSP.CompletionParams CreateCompletionParams(LSP.Location caret,
         protected static LSP.VSCompletionItem CreateCompletionItem(
             string text, LSP.CompletionItemKind kind, string[] tags,
             LSP.CompletionParams requestParameters, bool preselect = false,
-            string[]? commitCharacters = null)
+            string[]? commitCharacters = null, string? sortText = null)
             => new LSP.VSCompletionItem()
             {
                 FilterText = text,
                 InsertText = text,
                 Label = text,
-                SortText = text,
+                SortText = sortText ?? text,
                 InsertTextFormat = LSP.InsertTextFormat.Plaintext,
```

Comment:
oooh, what does this mean?

Reply:
Test helper - in most cases the SortText = insert text, but in some cases it isn't (e.g. date time, regex completions)

So I pass in the sorttext to the expected test item only if I expect it to be different.